compiler/rustc_target/src/spec/aarch64_nintendo_switch_linker_script.ld

```diff
@@ -0,0 +1,83 @@
+OUTPUT_FORMAT(elf64-littleaarch64)
+OUTPUT_ARCH(aarch64)
+ENTRY(_start)
+
+PHDRS
+{
+  text PT_LOAD FLAGS(5);
+  rodata PT_LOAD FLAGS(4);
+  data PT_LOAD FLAGS(6);
+  bss PT_LOAD FLAGS(6);
+  dynamic PT_DYNAMIC;
+}
+
+SECTIONS
+{
+  . = 0;
+
+  .text : ALIGN(0x1000) {
+    HIDDEN(__text_start = .);
+    KEEP(*(.text.jmp))
+
+    . = 0x80;
+
+    *(.text .text.*)
+    *(.plt .plt.*)
+  }
+
+  /* Read-only sections */
+
+  . = ALIGN(0x1000);
+
+  .module_name : { *(.module_name) } :rodata
+  
+  .rodata : { *(.rodata .rodata.*) } :rodata
+  .mod0 : {
+    KEEP(crt0.nso.o(.data.mod0))
+    KEEP(crt0.nro.o(.data.mod0))
+    KEEP(crt0.lib.nro.o(.data.mod0))
```

Comment:
What are those object files? Where can you get them from?

Reply:
Actually, I'm not sure these are necessary. cc @XorTroll (who I believe wrote the original linker script?)

Reply:
Yeah, this was just a mistake. I don't think these have existed for ages, I've removed them.

Reply:
These were object files generated from assembly in libtransistor (permissive OSS for building homebrew on the switch) for starting up the executable. They aren't needed under Rust's build system. As leo said he's removed them from the linker script, although that wasn't doing anything before.